src/Models/Attribute.php

```diff
@@ -17,6 +18,13 @@ protected static function booting()
         static::addGlobalScope(new OnlyProductAttributesScope);
     }
 
+    protected function filter(): CastsAttribute
+    {
+        return CastsAttribute::make(
+            get: fn ($value) => $value || in_array($this->code, config('rapidez.extra-filters')),
+        )->shouldCache();
+    }
```

Comment:
`Attribute` vs `CastsAttribute`? Did you check the link? https://laravel.com/docs/master/eloquent-mutators#defining-an-accessor

Reply:
Yeah see the thing is that I can't just use the class using its name `Attribute` because, well, the class we're on is already called that :)